<launch>

	<node pkg="course_agv_slam_task" type="extraction" name="extraction">
		<param name="range_threshold" value="1.0"/>
		<param name="radius_max_th" value="0.25"/>
		<param name="landMark_min_pt" value="3"/>
	</node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find course_agv_slam_task)/launch/course_agv.rviz" />
</launch>
